Added exploration documentation (#3703)
conveyordata/data-product-portal: docs/docs/concepts/explorations.md

---
id: explorations
title: Explorations
sidebar_position: 2
---

# Explorations

> **Explorations** are great when you want to answer a one-time answer, or need a personal sandbox to experiment.

Reviewer: ```suggestion
**Explorations** are great when you want to answer a one-time question, or need a personal sandbox to experiment.
```

> They do not allow you to share any data with others. And thus do not support [Output Ports](./output-ports) or [Technical

Reviewer: ```suggestion
They do not allow you to share any data with others. Hence they do not support [Output Ports](./output-ports) or [Technical
```

Assets](./technical-assets).

They do support [Input Ports](./input-ports) since these are the way to get access to data.

> And hence it's possible to track where data is used throughout the organisation, even for simple one-time questions.

Reviewer: ```suggestion
This enables you to track where data is used throughout the organisation, even for simple one-time questions.
```
